packages/initializer-jwt/src/index.ts

```diff
@@ -16,6 +18,18 @@ export interface JWTProps {
 const key = 'jwt'
 export const getProps = () => context.initializerProps<JWTProps>(key)
 
+export const resolveAuthCookieName = (
+  cookieNameConfig?: string | boolean
+): string | null => {
```

Comment:
before this whole thing, do
```
if(!cookieNameConfig){
  return null
}
```
this clears out null, false, and other falsy values

Reply:
also, instead of 
if ..
return
if ..
return

use proper if/else if/else
it clarifies the intent in a much better way